libsolidity/AST.h

```diff
@@ -156,6 +156,68 @@ class Declaration: public ASTNode
 	Declaration const* m_scope;
 };
 
+
+/**
+* Generic Parameter description used by @see FunctionDescription to return
```

Comment:
wow how did you notice that? Funny ... my editor should take care of these alignments normnally